homeassistant/components/iron_os/strings.json

```diff
@@ -0,0 +1,83 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "[%key:component::bluetooth::config::step::user::description%]",
+        "data": {
+          "address": "[%key:common::config_flow::data::device%]"
+        }
+      },
+      "bluetooth_confirm": {
+        "description": "[%key:component::bluetooth::config::step::bluetooth_confirm::description%]"
+      }
+    },
+    "abort": {
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "live_temperature": {
+        "name": "Tip temperature"
+      },
+      "voltage": {
+        "name": "DC Input Voltage"
```

Comment:
Let's try to omit the Unit of measurement here as someone could also set it to microvolts

Reply:
what unit of measurement?

Reply:
Voltage is a unit. but if the user sets it to microvolts, the name is incorrect. so maybe try to incorporate `potential`.

Looks like a nitpick, it is, but its just a thing we have with naming

Reply:
But voltage is not a unit of measeurement, that's how it's called in english and what this sensor measures. And the unit  of measurement for voltage is volt (or kVolt, mVolt, uVolt...).

Reply:
That is true now I think about it